katas/content/key_distribution/index.md

```diff
@@ -0,0 +1,136 @@
+# Quantum Key Distribution
+
+@[section]({
+    "id": "key_distribution__overview",
+    "title": "Overview"
+})
+
+Quantum key distribution is a type of quantum communication protocols that allow two parties to generate shared secret keys - random strings of bits known only to those two parties. These shared keys can then be used for a variety of different classical cryptographic protocols like encryption or authentication.
```

Comment:
```suggestion
Quantum key distribution is a type of quantum communication protocol that allow two parties to generate shared secret keys - random strings of bits known only to those two parties. These shared keys can then be used for a variety of different classical cryptographic protocols like encryption or authentication.
```

Reply:
```suggestion
Quantum key distribution is a type of quantum communication protocol that allows two parties to generate shared secret keys - random strings of bits known only to those two parties. These shared keys can then be used for a variety of different classical cryptographic protocols like encryption or authentication.
```

Reply:
Good point, but then we need to change the verb too